packages/protocol/migrationsConfig.js

```diff
@@ -26,6 +26,22 @@ const DefaultConfig = {
     maxVotesPerAccount: 3,
     electabilityThreshold: 1 / 100,
   },
+  epochRewards: {
+    targetVotingYieldParameters: {
+      initial: 5 / 100,
+      max: 2 / 10,
+      adjustmentFactor: 1 / 365,
+    },
+    rewardsMultiplierParameters: {
+      max: 2,
+      adjustmentFactors: {
+        underspend: 1 / 2,
+        overspend: 5,
+      },
+    },
+    targetVotingGoldFraction: 2 / 3,
+    maxValidatorEpochPayment: '1000000000000000000',
```

Comment:
Did we decide to increase this from 75k to 100k cUSD?

Reply:
Fixed